src/main/java/com/rybka/config/MapSearchUtil.java

```diff
@@ -0,0 +1,12 @@
+package com.rybka.config;
+
+import java.util.Map;
+
+public class MapSearchUtil {
+    public static <T> Map.Entry<String, T> retrieveMapValue(Map<String, T> map, String searchItem, RuntimeException runtimeException) {
+        return map.entrySet().stream()
+                .filter(item -> searchItem.equals(item.getKey()))
+                .findFirst()
+                .orElseThrow(() -> runtimeException);
```

Comment:
create new instance of exception here. 
don't see reason to put it into the parameters.

Reply:
fixed